controllers/infinispan_service_controller.go

```diff
@@ -466,8 +465,7 @@ func computeServiceExternal(ispn *ispnv1.Infinispan) *corev1.Service {
 
 // computeSiteService compute the XSite service
 func computeSiteService(ispn *ispnv1.Infinispan) *corev1.Service {
-	lsPodSelector := PodLabels(ispn.Name)
-	lsPodSelector[consts.CoordinatorPodLabel] = strconv.FormatBool(true)
```

Comment:
@pruivo coordinator pod labeling seems to be removed also

Reply:
it is not required for cross-site.

Reply:
I talked about to remove this logic. Do we really need it after GossipRouter Implementation?
https://github.com/infinispan/infinispan-operator/blob/6381d8b0fd4aca538fab2c19c97a40d6374a19b9/controllers/xsite.go#L17-L26

Reply:
right! I've removed it.